features/daemon.feature

```diff
@@ -83,19 +83,32 @@ Feature: Pro Upgrade Daemon only runs in environments where necessary
         """
         inactive
         """
+
         # verify it stays on when configured to do so
         When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
         """
         ua_config:
             poll_for_pro_license: true
         """
+        # Turn on memory accounting
+        When I run `sed -i s/#DefaultMemoryAccounting=no/DefaultMemoryAccounting=yes/ /etc/systemd/system.conf` with sudo
```

Comment:
@orndorffgrant Is there a specific reason why gcp generic machines do not default to yes for *DefaultMemoryAccounting*?

Reply:
Focal ones have it on by default, but bionic doesn't. I didn't document the other series because I needed to add this line regardless.
There may be a specific reason for the defaults, but honestly I don't know :)